layouts/rss.xml

```diff
@@ -11,13 +11,16 @@
 {{- $pages = $pages | first $limit -}}
 {{- end -}}
 {{- printf "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"yes\"?>" | safeHTML }}
-<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:webfeeds="http://webfeeds.org/rss/1.0">
+<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:content="http://purl.org/rss/1.0/modules/content/">
   <channel>
     <title>{{ if eq  .Title  .Site.Title }}{{ .Site.Title }}{{ else }}{{ with .Title }}{{.}} on {{ end }}{{ .Site.Title }}{{ end }}</title>
     <link>{{ .Permalink }}</link>
     <description>Recent content {{ if ne  .Title  .Site.Title }}{{ with .Title }}in {{.}} {{ end }}{{ end }}on {{ .Site.Title }}</description>
-    {{- with .Site.Params.customFavicon -}}
-    <webfeeds:icon>{{ . | absURL }}</webfeeds:icon>
+    {{- with .Site.Params.siteLogoImage -}}
+    <image>
+      <url>{{ . | absURL }}</url>
+      <link>{{ . | absURL }}</link>
+    </image>
```

Comment:
This is probably not equivalent to  `<webfeeds:icon>` as it does not signal that it is an icon? The original syntax will allow RSS readers such as Feedly to use the specified image as the icon. See https://blog.haysc.tech/hexo-feed-setup/#%E8%87%AA%E5%88%B6-XML-%E6%A8%A1%E6%9D%BF%EF%BC%9A%E9%9D%A2%E5%90%91-Feedly-%E4%BC%98%E5%8C%96.